spec/ref_arch_setup/install_spec.rb

```diff
@@ -45,14 +47,14 @@
         it "returns true and outputs information about the bolt run" do
```

Comment:
We aren't checking the output, so we should drop the part about outputs information about the bolt run.